crates/orchestrator/src/alerts/aws_sns/mod.rs

```diff
@@ -10,9 +10,13 @@ pub struct AWSSNS {
 
 impl AWSSNS {
     /// To create a new SNS client
-    pub async fn new() -> Self {
-        let sns_region = get_env_var_or_panic("AWS_SNS_REGION");
-        let config = aws_config::from_env().region(Region::new(sns_region)).load().await;
+    pub async fn new(config: &SdkConfig) -> Self {
+        AWSSNS { client: Client::new(config) }
+    }
+
+    /// To create a new SNS client from the environment variables
+    pub async fn new_from_env() -> Self {
```

Comment:
if this function isn't being used, let's delete

Reply:
deleted.